src/tools/dart2native.md

```diff
@@ -56,6 +63,39 @@ You can then run the app using the [`dartaotruntime` command][]:
 $ dartaotruntime bin/main.aot
 ```
 
+## Known limitations
+
+The initial (Dart 2.6) version of `dart2native` has some known limitations:
+
+No cross-compilation support ([issue 28617][])
+: The compiler supports creating machine code only for
+  the operating system it’s running on.
+  You need to run the compiler three times —
+  on macOS, Windows, and Linux —
+  to create executables for all three operating systems.
+  A workaround is to use a CI (continuous integration) provider
+  that supports all three operating systems.
+
+No signing support ([issue 39106][])
+: The format of the executables isn’t compatible with
+  standard signing tools such as codesign and SignTool.
+
+No support for dart:mirrors and dart:developer
+: The code compiled by `dart2native` can use all of the other libraries
+  that the Dart VM supports.
```

Comment:
On the page linked to here, the term 'Dart VM' isn't really used

Reply:
That page should explain that JIT means — it's very unclear. I think we meant to do that but forgot.